solution/backend/createsu.py

```diff
@@ -9,8 +9,10 @@ def handler(self, *args, **options):
 
     from django.contrib.auth.models import Group, User
 
-    e_regs_admin_group, _ = Group.objects.get_or_create(name='EREGS-ADMIN')
-    e_regs_reader_group, _ = Group.objects.get_or_create(name='EREGS-READER')
+    e_regs_admin_group, _ = Group.objects.get_or_create(name='EREGS_ADMIN')
+    e_regs_reader_group, _ = Group.objects.get_or_create(name='EREGS_READER')
+    e_regs_editor_group, _ = Group.objects.get_or_create(name='EREGS_EDITOR')
+    e_regs_manager_group, _ = Group.objects.get_or_create(name='EREGS_MANAGER')
 
     if not User.objects.filter(username=os.environ.get('DJANGO_ADMIN_USERNAME')).exists():
```

Comment:
do you want to just set this to check if they are "EREGS_ADMIN".  Since we have the multiple groups and only they should be super user it might make sense to change that now